tests/longevity/results/1.0.0.md

```diff
@@ -0,0 +1,234 @@
+# Results for v1.0.0
```

Comment:
Shall we have a `results/1.0.0/` directory, similar to Kate's recent structure changes in her PR?